packages/foundations/package.json

```
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "npm-run-all build:*",
     "build:01_style-dictionary": "node style-dictionary.js && npm run copy:scss",
-    "build:02_css": "sass --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ build/scss/:build/css/ --future-deprecation=import",
+    "build:02_css": "sass --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ build/scss/:build/css/",
```

```suggestion
    "build:02_css": "sass --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ build/scss/:build/css/ --future-deprecation=import",
```

why remove it? It might be a good reminder in case somebody still uses it.

---

https://github.com/db-ui/mono/pull/673#discussion_r1140391061